marine.c

```diff
@@ -783,6 +784,15 @@ WS_DLL_PUBLIC int init_marine(void) {
     return 0;
 }
 
+WS_DLL_PUBLIC int init_marine(void) {
+    if (marine_loaded) {
+        return -2;
```

Comment:
Export it to a const and expose it in `marine.h`, so clients will be able to use that